src/com/company/Batsman.java

```diff
@@ -0,0 +1,15 @@
+package com.company;
+
+public class Batsman {
```

Comment:
is this class used anywhere?

Reply:
Sorry! Was earlier trying to use it but then found that it is appropriate to use player and forgot to delete it.